Lib/test/test_locale.py

```diff
@@ -565,9 +568,10 @@ def test_strcoll_3303(self):
         self.assertRaises(TypeError, locale.strcoll, "a", None)
         self.assertRaises(TypeError, locale.strcoll, b"a", None)
```

Comment:
Instead of editing tests inside, we usually do:
```rust
    # TODO: RUSTPYTHON
    if sys.platform == "win32":
        test_defaults_UTF8 = unittest.expectedFailure(test_defaults_UTF8)
```
Searching for `unittest.expectedFailure(` will show examples